### PR TITLE
Hiba

### PR DESCRIPTION
**Pull request by HibaBenAribi**
I worked on Marina's code, So I suppose her PR should be merged first.

### CODE REVIEW

File: VariantSurvival_package/R/VariantSurvival.R [hunk 1]
```diff
@@ -1,425 +1,403 @@
...
+                   br(),
+                   br(),
+                   span(shiny::tags$i(h3("Cox regression table")),
+                        style="color:#045a8d"),
+                   DT::dataTableOutput("table3")
+                   ),
+                 mainPanel(
+                   span(shiny::tags$i(h2("Kaplan–Meier")), 
+                   shinycssloaders::withSpinner(plotOutput("plot_km"))
+                   )
```
Comment (msarrias) [inline VariantSurvival_package/R/VariantSurvival.R:83]: @HibaBenAribi  closing parentheses are missing.

Comment (HibaBenAribi) [reply VariantSurvival_package/R/VariantSurvival.R:83]: corrected

File: VariantSurvival_package/R/VariantSurvival.R [hunk 2]
```diff
@@ -1,425 +1,403 @@
...
+}
+
+
+#' `get_disease_gene_list` parses an existing excel 
+#' file containing all known target genes associated
+#' with the selected disease.
+#' @param input_disease: input disease selection
+#' @return  genes_list
+get_disease_gene_list <- function(input_disease) {
+  disease_gene_select <- disease_gene[,input$disease_n]
```
Comment (msarrias) [inline VariantSurvival_package/R/VariantSurvival.R:297]: what is disease_gene? you're referencing a df that's not defined within the function. Better pass it as an argument.

Comment (HibaBenAribi) [reply VariantSurvival_package/R/VariantSurvival.R:297]: disease_gene is dataframe = excel file containing genes list
L17: disease_gene <- read_excel("disease_gene.xlsx")
the command disease_gene[,input$disease_n] should extract the column = input$disease_n

**Comment by Krannich479** [comment]
I'll review changes and collision here once we have PR #10 in place.

**Pull request closed without merging**